### ISSUE TITLE
Dash for Mac

### ISSUE THREAD

**Issue by BryanGFS** [nevadascout/Dash#75]
Is there anyway we can compile it for mac also?

I do a lot of dev work on my mac and the cross load it to my windows PC to test in real time, It's great for when I'm in class too.

**Comment by nevadascout** [comment]
At the current time, Dash is written in C# using the .NET framework with some custom controls. This means that in it's current state, Dash cannot be built for anything other than Windows.

However, in the future it could potentially be ported to Mono and this would then allow for cross-platform builds.

I have no plans to do this until all of the core features are finished an implemented in version 2.0. This means that version 3.0 of Dash could be cross-platform, but I can't confirm anything.

Right now, it's just me working on it by myself and I'm overworked as it is!: )

**Comment by trdwll** [comment]
C# (.NET/.exe Files) can be ran with Wine (https://www.winehq.org/) on other OS'. However, rebuilding Dash with C/C++ or some other language (maybe Python) could make it a better overall tool.

**Comment by nevadascout** [comment]
Yep, potentially could use Xamarin/mono to run Dash cross-platform as well
